### DISCUSSION TITLE
CVE-2022-42889 in version 2.1.0 and 2.1.2

### DISCUSSION THREAD

**Discussion by harpreet86** [dropwizard/dropwizard#6116] [General]
There is a vulnerability(CVE-2022-42889) reported related Apache commons text and as per the current recommendation we need to migrate to version 1.10.0 of Apache commons text.

We found that in our project this dependency is coming from dropwizard.
As we are using dropwizard 2.1.0 and 2.1.2, which are using Apache commons text 1.9.0.
We also found that in the latest version(2.1.4) of dropwizard, the version of Apache commons text has been migrated to 1.10.0

Now the question is that were the versions 2.1.0 and 2.1.2 were impacted by this vulnerability, this will help us in planning/prioritizing our dropwizard migration.

@joschi please suggest, tagging considering the criticality of the issue.

**Comment by joschi** [comment]
See https://github.com/dropwizard/dropwizard/discussions/6094#discussioncomment-3909525.

> tagging considering the criticality of the issue

No, not really.

📝 [Is CVE-2022-42889 the next Log4Shell? Not really.](https://www.chainguard.dev/unchained/is-cve-2022-42889-the-next-log4shell-not-really)
